src/vs/workbench/contrib/files/browser/files.contribution.ts

```diff
@@ -367,6 +367,11 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('files.simpleDialog.enable', "Enables the simple file dialog for opening and saving files and folders. The simple file dialog replaces the system file dialog when enabled."),
 			'default': false
 		},
+		'files.simpleDialog.currentDirectory': {
```

Comment:
Maybe I miss something, but how is this setting only for simple file dialog? If there is a desire for this, I would make it work in native and simple file dialog.

Reply:
Thank you for the feedback! My initial intention was to make changes specifically for the simple file dialog, so I didn't anticipate these changes affecting other components or the native file dialog.

I tested it with the native file dialog, and it works with that one as well:

https://github.com/user-attachments/assets/e5725d6d-903f-40f7-98df-98f9ddcb1206

I'll update the option name to better reflect its current functionality.

That said, any further enhancements are appreciated.